cmd/podmanV2/manifest/manifest.go

```diff
@@ -0,0 +1,37 @@
+package manifest
+
+import (
+	"github.com/containers/libpod/cmd/podmanV2/registry"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/spf13/cobra"
+)
+
+var (
+	manifestDescription = "Creates, modifies, and pushes manifest lists and image indexes."
+	manifestCmd         = &cobra.Command{
+		Use:               "manifest",
+		Short:             "Manipulate manifest lists and image indexes",
+		Long:              manifestDescription,
+		TraverseChildren:  true,
+		PersistentPreRunE: preRunE,
```

Comment:
```suggestion
```
As of https://github.com/containers/libpod/pull/5848 this is no longer required.